### PR TITLE
Convert official pipeline to1ES template

### PR DESCRIPTION
**Pull request by phil-allen-msft**
Convert official pipeline to1ES template

### CODE REVIEW

File: azure-pipelines/azure-pipeline.microbuild.before.yml [hunk 1]
```diff
@@ -1,10 +1 @@
-steps:
-
-- task: MicroBuildSigningPlugin@3
-  inputs:
-    signType: $(SignType)
-    zipSources: false
-  displayName: Install MicroBuild Signing Plugin
-
-- task: MicroBuildSwixPlugin@4
-  displayName: Install MicroBuild Swix Plugin
```
Comment (alexgav) [inline azure-pipelines/azure-pipeline.microbuild.before.yml:10]: I didn't see where this went - I'm guessing parent template we are extending?

**Review by alexgav — APPROVED** [review]
:shipit:

File: azure-pipelines/azure-pipeline.microbuild.before.yml (hunk 1 above)
Comment (tlmii) [reply azure-pipelines/azure-pipeline.microbuild.before.yml:10]: It ended up here:

https://github.com/aspnet/LibraryManager/blob/77d22d5bcebe4b2773577a4076adc5ba01a2f4a3/azure-pipelines/official.yml#L38-L44

File: azure-pipelines/azure-pipeline.microbuild.before.yml [hunk 2]
```diff
@@ -1,10 +1 @@
...
+steps: []
```
Comment (tlmii) [inline azure-pipelines/azure-pipeline.microbuild.before.yml:1]: Should we just delete this file (and the reference to it)? I guess no harm in leaving it if its needed in the future. Just seems odd having an (essentially) empty file here.

File: azure-pipelines/official.yml [hunk 3]
```diff
@@ -11,8 +11,65 @@ variables:
...
+          mb:
+            signing:
+              enabled: true
+              signType: $(SignType)
+              zipSources: false
+            swix:
+              enabled: true
+          outputParentDirectory: $(Build.ArtifactStagingDirectory)
+          outputs:
+          - ${{ if eq(variables['system.collectionId'], '011b8bdf-6d56-4f87-be0d-0092136884d9') }}:
```
Comment (tlmii) [inline azure-pipelines/official.yml:47]: Will this ever not be true for this yaml (I assume official only runs in the internal pipeline)?

**Review by tlmii — APPROVED** [review]
Some comments, but nothing blocking. Looks good!

**Pull request merged**